PortalForecasts.R

```diff
@@ -156,12 +156,13 @@ forecastall <- function(abundances,level,weather_data,weatherforecast) {
       model_aic = 1e6
     } else {
       model=tsglm(species_abundance,model=list(past_obs=1,past_mean=12),distr="nbinom")
```

Comment:
Pretty sure this 12 is different than `num_forecast_months`. from ?tsglm  

past_mean
integer vector giving the previous conditional means to be regressed on. This is a vector with the elements j[1],…,j[q] (see Details). If omitted, or of length zero, there will be no regression on previous conditional means.